pkg/apis/networking/types.go

```diff
@@ -336,19 +338,45 @@ type HTTPIngressRuleValue struct {
 	// options usable by a loadbalancer, like http keep-alive.
 }
 
-// HTTPIngressPath associates a path regex with a backend. Incoming urls matching
-// the path are forwarded to the backend.
+// PathType represents the type of path referred to by a HTTPIngressPath.
+type PathType string
+
+const (
+	// PathTypeExact matches the URL path exactly.
+	PathTypeExact = PathType("Exact")
+
+	// PathTypePrefix matches based on a URL path prefix split by '/'. Matching
+	// is done on a path element by element basis. A path element refers is the
+	// list of labels in the path split by the '/' separator. A request is a
+	// match for path p if every p is an element-wise prefix of p of the request
+	// path. Note that if the last element of the path is a substring of the
+	// last element in request path, it is not a match (e.g. /foo/bar matches
+	// /foo/bar/baz, but does not match /foo/barbaz).
```

Comment:
clarify trailing slash behavior?
* does `path: /foo` match a request of `/foo/`?
* does `path: /foo/` match a request of `/foo`?

Reply:
Added some clarification to the docs, thanks for catching this!